test/Microsoft.ML.Benchmarks/Microsoft.ML.Benchmarks.csproj

```diff
@@ -4,6 +4,8 @@
     <OutputType>Exe</OutputType>
     <LangVersion>7.2</LangVersion>
     <StartupObject>Microsoft.ML.Benchmarks.Program</StartupObject>
+    <TargetFramework Condition="'$(UseIntrinsics)' != 'true'">netcoreapp2.1</TargetFramework>
```

Comment:
This shouldn't be required since it is handled here:

https://github.com/dotnet/machinelearning/blob/061788d73351ce0560e67d146cd997f806713ee9/test/Directory.Build.props#L5-L8